docker/lib/dependabot/docker/requirement.rb

```diff
@@ -11,21 +11,24 @@ module Docker
     # Lifted from the bundler package manager
     class Requirement < Dependabot::Requirement
       extend T::Sig
+      extend T::Helpers
```

Comment:
I'm not sure you need this `extend`? What feature of Sorbet do you need from it?